kratos/tests/cpp_tests/processes/test_apply_ray_casting_interface_recognition_process.cpp

```diff
@@ -0,0 +1,103 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Daniel Diez
+//
+
+// Project includes
+#include "testing/testing.h"
+#include "processes/apply_ray_casting_interface_recognition_process.h"
+
+namespace Kratos::Testing {
+    KRATOS_TEST_CASE_IN_SUITE(ApplyRayCastingWithInterfaceProcess3D, KratosCoreFastSuite)
```

Comment:
```suggestion
    KRATOS_TEST_CASE_IN_SUITE(ApplyRayCastingInterfaceRecognitionProcess3D, KratosCoreFastSuite)
```
Very minor but I'd call the test as the class to ease browsing the code in case of need.

Reply:
done!